doc/frequent-itemsets.rst

```diff
@@ -0,0 +1,62 @@
+Frequent Itemsets
+=================
+
+.. figure:: icons/frequent-itemsets.png
+
+Finds frequent itemsets in the data.
+
+Signals
+-------
+
+#### Inputs:
+
+-  **Data**
+
+Data set
```

Comment:
So we are again changing the layout? Ok. What is the difference between two, four, three spaces?